pkg/cmd/grafana-server/server.go

```diff
@@ -193,7 +195,9 @@ func (g *GrafanaServerImpl) Shutdown(reason string) {
 	g.shutdownFn()
 
 	// wait for child routines
-	g.childRoutines.Wait()
+	if err := g.childRoutines.Wait(); err != nil {
+		g.log.Debug("Error waiting on child routines", "err", err)
```

Comment:
Error instead of debug and maybe "Failed waiting for services to shutdown" to give some more context?